go.sum

```diff
@@ -29,6 +29,10 @@ github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142/go.mod h1:F5haX7
 github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f h1:lBNOc5arjvs8E5mO2tbpBpLoyyu8B6e44T7hJy6potg=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
+github.com/crazycs520/parser v0.0.0-20190801014535-8fddad2565dd h1:sELKj3l9keUcFBZGKsKC7vK9W+zofZiIDTSlDvHpSc0=
+github.com/crazycs520/parser v0.0.0-20190801014535-8fddad2565dd/go.mod h1:xLjI+gnWYexq011WPMEvCNS8rFM9qe1vdojIEzSKPuc=
+github.com/crazycs520/parser v0.0.0-20190801015123-1039a56acb0c h1:OCvqcYqI/9Aqula24IjLMHGcsDL/xOhsU3Mh3gXBS3A=
+github.com/crazycs520/parser v0.0.0-20190801015123-1039a56acb0c/go.mod h1:xLjI+gnWYexq011WPMEvCNS8rFM9qe1vdojIEzSKPuc=
```

Comment:
?

Reply:
Need merge parser first, then I will update the go.mod.